packages/core/src/database/insert-into.ts

```diff
@@ -70,12 +77,15 @@ export const buildInsertIntoWithPool =
           insertingKeys.map((key) => convertToPrimitiveOrSql(key, data[key] ?? null)),
           sql`, `
         )})
-        ${conditionalSql(
-          onConflict,
-          ({ fields, setExcludedFields }) => sql`
-            on conflict (${sql.join(fields, sql`, `)}) do update
-            set ${setExcluded(...setExcludedFields)}
-          `
+        ${conditionalSql(onConflict, ({ fields, setExcludedFields }) =>
+          fields && setExcludedFields
```

Comment:
looks like `ignore` is not in use. would like to double check the usage

Reply:
updated